src/main/java/com/example/kaizenchat/controller/DuoChatController.java

```diff
@@ -151,7 +151,7 @@ public ResponseEntity<Map<String, Object>> getLastMessages(@Valid @RequestBody L
                 messages = messageService.getLastMessages(request.getChatId(), request.getTime(), GET_MESSAGES_LIMIT);
             return ResponseEntity.ok().body(Map.of("messages", messages));
         }else
-            return ResponseEntity.status(HttpStatus.NOT_FOUND).body(Map.of("message", "not member of chat"));
+            throw new UserNotFoundInChatException();
```

Comment:
[LINE:154] please add message to the exception "not member of chat". It is needed to pass this message to the client